mitosheet/css/sitewide/colors.css

```diff
@@ -30,4 +30,10 @@
 
     /* Color for success message */
     --mito-green: #10710D;
+
+    /* Colors used in recon */
+    --mito-recon-created-background-color: #96D394;
+    --mito-recon-created-text-color: #10710D;
+    --mito-recon-modified-background-color: #EDDAA2;
+    --mito-recon-modified-text-color: #ECBB4A;
```

Comment:
One thing on colors: I think columns should have colors alternating per row, even if they are highlighted green. 

E.g. we need to add `--mito-recon-created-text-color-light` or something, and odd rows become this color, while odd rows are `--mito-recon-created-text-color`. Otherwise, it's much harder to parse the data.